testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-spark-iceberg-jdbc-catalog/create-table.sql

```diff
@@ -0,0 +1,20 @@
+/**
+  Table definition in Iceberg repository:
+  https://github.com/apache/iceberg/blob/master/core/src/main/java/org/apache/iceberg/jdbc/JdbcUtil.java
+ */
+CREATE TABLE iceberg_namespace_properties (
+    catalog_name VARCHAR(255) NOT NULL,
+    namespace VARCHAR(255) NOT NULL,
+    key VARCHAR(5500),
```

Comment:
It looks like the master merged iceberg PR has updated these table column names to avoid conflicts with reserved SQL keywords:
Posting it here for reference:
https://github.com/apache/iceberg/pull/3275/files#diff-1daee532e0ff506f3e2fa8186deb993eec463498c6aea8c814dcd604891aa1e1R88

Reply:
The column name looks still `key` https://github.com/apache/iceberg/pull/3275/files#diff-1daee532e0ff506f3e2fa8186deb993eec463498c6aea8c814dcd604891aa1e1R80. Did you post wrong URL?